### PR TITLE
fix report when no exposure summary found

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix?

* Ticket: #
* Funded by: DFAT
* Description: 
  * another fix for raster on raster analysis type  when generating report (check #4907 )

### Checklist:

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: safe/impact_function/multi_exposure_wrapper.py [hunk 1]
```diff
@@ -985,12 +986,14 @@ def run(self):
             dict_of_analysis_summary_id[exposure_key] = (
                 impact_function.analysis_impacted.id())
 
-            # eq raster on pop raster might not having this
-            if impact_function.exposure_summary:
-                dict_of_exposure_summary_path[exposure_key] = full_layer_uri(
-                    impact_function.exposure_summary)
-                dict_of_exposure_summary_id[exposure_key] = (
-                    impact_function.exposure_summary.id())
+            # Exposure summary layer might not exist for raster on raster
```
Comment (Gustry) [inline safe/impact_function/multi_exposure_wrapper.py]: The exposure summary layer does not exist for: exposure continuous raster layer (whatever the hazard type).
We still have it for exposure classified raster layer.

File: safe/impact_function/multi_exposure_wrapper.py [hunk 2]
```diff
@@ -1211,6 +1214,7 @@ def generate_report(
 
                 if exposure_groups:
                     for exposure_group in exposure_groups:
+                        analysis_layer = None
```
Comment (Gustry) [inline safe/impact_function/multi_exposure_wrapper.py]: Better to use the same wording as in the impact_function `impact_layer`? https://github.com/inasafe/inasafe/blob/develop/safe/impact_function/impact_function.py#L634

**Comment by myarjunar** [comment]
thanks @Gustry  I have updated the code

**Comment by Gustry** [comment]
LGTM, thanks !

**Pull request merged**